internal/controller/controllers/controller_event_wrapper.go

```diff
@@ -101,14 +101,14 @@ func (c *controllerEventsWrapper) NotifyKubeApiClusterEvent(clusterID strfmt.UUI
 	c.crdEventsHandler.NotifyClusterDeploymentUpdates(cluster.KubeKeyName, cluster.KubeKeyNamespace)
 }
 
-func (c *controllerEventsWrapper) NotifyKubeApiHostEvent(clusterID strfmt.UUID, hostID strfmt.UUID) {
-	host, err := common.GetHostFromDB(c.db.Unscoped(), clusterID.String(), hostID.String())
+func (c *controllerEventsWrapper) NotifyKubeApiHostEvent(infraEnvID string, hostID string) {
```

Comment:
there is not real issue to use `strfmt.UUID` here as long as you access pointers safely

Reply:
Done